client/testconf/fake-gcs.conf.yml

```diff
@@ -1,3 +0,0 @@
-type: FAKE-GCS
```

Comment:
Why changing, sounds like we remve some test cases which right now are uncovered? 🤔

Reply:
These were not utilized. I haven't tracked them down but my they are left overs from the migration.

Reply:
👍🏽